### PR TITLE
Feature/character development

### PR DESCRIPTION
**Pull request by yoerivanhoof**
This Pull Request (PR) adds functionality to the game according to User Story #XXX

This PR...

* [x] has added functionality.
* [x] complies with the code style of the repository.
* [ ] adds new unit tests where necessary.
* [x] is up to date with the `dev` branch.

Also, I have...

* [x] Updated Taiga.

Please merge with care!

### CODE REVIEW

File: memecity.game/ExpComponent.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+std::string ExpComponent::COMPONENT_TYPE = "ExpComponent";
+
+ExpComponent::ExpComponent(Entity* e): Component(e)
+{
+}
+
+std::string ExpComponent::get_type()
+{
+	return std::string("ExpComponent");
```
Comment (tvbronswijk) [inline memecity.game/ExpComponent.cpp]: het idee is dat je `COMPONENT_TYPE` returned

File: memecity.game/ExpComponent.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#pragma once
```
Comment (tvbronswijk) [inline memecity.game/ExpComponent.h]: `pragma once` 😠

File: memecity.game/ExpEventArgs.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <EventArgs.h>
+class ExpEventArgs :
```
Comment (tvbronswijk) [inline memecity.game/ExpEventArgs.h]: waarom is dit een `class` ipv een `struct`?

File: memecity.game/ExpSystem.cpp [hunk 4]
```diff
@@ -0,0 +1,50 @@
+#include "ExpSystem.h"
+#include <string>
```
Comment (tvbronswijk) [inline memecity.game/ExpSystem.cpp]: `include` in de source file 😠

File: memecity.game/GameManager.cpp [hunk 5]
```diff
@@ -1,11 +1,27 @@
...
 {
 	
 	if (multimedia_manager->init())
 	{
 		multimedia_manager->play_background_music("bgm.mp3", 50);
+		texture = multimedia_manager->get_texture("BlikBier.bmp");
+		text = multimedia_manager->get_text_texture("Test", "Blazed.ttf", 50, { 255,10,10 });
+		text->translate({ 100.0f, 100.0f });
+
+		Entity* player = em.create_entity();
```
Comment (tvbronswijk) [inline memecity.game/GameManager.cpp]: voorbeeldcode?

File: memecity.game/GameManager.h [hunk 6]
```diff
@@ -9,6 +10,9 @@ class GameManager : public MemeEngine
 private:
 	std::unique_ptr<CityGenerator> city_generator;
 	std::unique_ptr<EntityManager> entity_manager;
+	std::shared_ptr<Texture> texture;
```
Comment (tvbronswijk) [inline memecity.game/GameManager.h]: dit is er uit gehaald, is niet meer nodig aub

File: memecity.game/InputSystem.h [hunk 7]
```diff
@@ -4,17 +4,18 @@
 #include "../memecity.engine.ecs/System.h"
 #include "../memecity.engine.ecs/EntityManager.h"
 
+class Event;
```
Comment (tvbronswijk) [inline memecity.game/InputSystem.h]: waarom gebruik je forward declarations ipv include?

**Review by tvbronswijk — CHANGES_REQUESTED** [review]

File: memecity.game/ExpComponent.cpp (hunk 1 above)
Comment (yoerivanhoof) [reply memecity.game/ExpComponent.cpp]: updated

**Review by tvbronswijk — APPROVED** [review]

**Pull request merged**